packages/helm/symphony/templates/symphony.yaml

```diff
@@ -268,6 +268,10 @@ spec:
                 x-kubernetes-preserve-unknown-fields: true
               siteId:
                 type: string
+              metadata:
```

Comment:
Inform Shawn to align this change with ARM specs.

Reply:
And make sure this is refreshed with code-gen once the above annotation is removed